NEWS.md

```diff
@@ -1,3 +1,8 @@
+# usethis 0.1.0.9000
+
+* Adding simple newlines for readability in `use_github` and fixing
```

Comment:
I think this change is small enough and reflects stuff that's only in the devel version that there's no need for a news bullet.